app/views/register/company/confirmCompanyDetails.scala.html

```diff
@@ -33,9 +33,9 @@
 
         @components.heading("companyAddress.heading")
 
-        <p>@name</p>
-
+        @name,
         @components.tolerant_address("companyAddress", address)
+        <br />
```

Comment:
is there another way of doing it other than a break tag?

Reply:
If it's just for adding a space you want to wrap this in a div with the `form-group` class. That should probably go in the `tolerant_address` helper.